src/modelmanager.cpp

```diff
@@ -82,6 +83,12 @@ void ModelManager::startWatcher() {
         watcherStarted = true;
         monitor = std::move(t);
     }
+
+    sequenceViewer->startWatcher();
+}
+
+void ModelManager::startSequenceWatcher() {
```

Comment:
Is this method used?

Reply:
In tests.

Reply:
Can we then either use it instead of calling `sequenceViewer->startWatcher();` 
or add it in mocked class so we don't have code unused in main logic?